frontend/src/hooks/useIsAdmin.js

```diff
@@ -0,0 +1,6 @@
+const useIsAdmin = () => {
```

Comment:
You can create 1 file maybe name as `useLocalStorage`
And put them there:
- frontend/src/hooks/useIsAdmin.js
- frontend/src/hooks/useIsUser.js